docs/GCSBucketCreate-action.md

```diff
@@ -1,3 +1,40 @@
-## Google Cloud Storage Create
+# Google Cloud Storage Create
 
-This plugin is used for creating directories on Google Cloud Storage (GCS).
+Description
```

Comment:
unrelated to this PR : ideally these plugins should be named GCS Object Create/Delete, GCS Bucket create/delete is misleading as we are not creating/deleting buckets.

Reply:
yeah... I also thought they just create/delete buckets at first. but renaming break compatibility. I think it's ok as long as the label is correct.